pymultimatic/model/mapper.py

```diff
@@ -291,7 +331,7 @@ def _map_function(
     if not target_high:
         target_high = conf.get("temperature_setpoint", None)
         if not target_high:
-            target_high = conf.get("temperatureSetpoint", None)
+            target_high = conf.get("setpoint", None)
```

Comment:
you should add a new `if not target_high` instead, otherwise it will break multimatic in some cases (as far as I remember, this is for a room)